### ISSUE TITLE
Fatal error after fresh installation, error with database login

### ISSUE THREAD

**Issue by zodit** [PrestaShop/PrestaShop#19972]
**Description and steps to reproduce the bug**

PrestaShop version: 1.7.6.5
PHP version: 7.2

Fresh installation.
The hosting is ok, I checked the compatibility with phppsinfo.php provided by Prestashop.
The installation process completed sucessfully.
I deleted the "install" folder as requested.
I renamed "admin" folder to access back office.

**Fatal Error**
The website outputs blank page both front office and back office.
Error log is the following:

PHP Fatal error:  Uncaught Symfony\Component\Debug\Exception\FatalThrowableError: Cannot use object of type Symfony\Component\HttpFoundation\ParameterBag as array in /MYROOT/src/Adapter/Security/Admin.php:91
Stack trace:
#0 /MYROOT/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/EventDispatcher.php(212): PrestaShop\PrestaShop\Adapter\Security\Admin->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\ContainerAwareEventDispatcher))
#1 /MYROOT/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/EventDispatcher.php(44): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent))
#2 /MYROOT/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php(127): Symfony\Component\EventDispatcher\EventDispatcher->dispatch('kernel.requ in /home2/wwwlovefactory/public_html/src/Adapter/Security/Admin.php on line 91

**Action taken and effect**

I wiped the var/cache folder. New errors appeared in log:

PHP Fatal error:  Uncaught PDOException: SQLSTATE[HY000] [1045] Access denied for user 'my_admin'@'localhost' (using password: YES) in /MYROOT/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43
Stack trace:
#0 /MYROOT/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php(43): PDO->__construct('mysql:host=127....', 'my_admin', '&3%%_16k}(x@H', Array)
#1 /MYROOT/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOMySql/Driver.php(41): Doctrine\DBAL\Driver\PDOConnection->__construct('mysql:host=127....', 'my_admin', '&3%%_16k}(x@H', Array)
#2 /MYROOT/vendor/doctrine/dbal/lib/Doctrine/DBAL/Connection.php(360): Doctrine\DBAL\Driver\PDOMySql\Driver->connect(Array, 'my_admin', '&3%%_16k}(x@H', Array)
#3 /MYROOT/vendor/doctrine/dbal/lib/Doctrine/DBAL/Connection.php(942): Doctrine\DBAL\Connection->connect()
#4 /MYROOT/src/PrestaShopBund in /MYROOT/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php on line 103

**Bug found**

I noticed that the password in the log was different, during installation process it was used to generate the database tables but then it was saved wrongly in app/config/parameters.php 
Real password is: &3%_16k}(x@H
Mistaken password is: &3%%_16k}(x@H

In short: a **%** character was added, i ignore what did it. I fixed the configuration file and now it is working, still this bug could happen again using passwords with characters i used.

**Comment by Progi1984** [comment]
Thanks for your report. I have fixed this in #19850. It's is in QA side.

**Comment by Progi1984** [comment]
We are aware of this issue, it is already in our debug backlog.This issue is a duplicate of #12556.To be informed when it's fixed, please subscribe to the issue mentioned above.Thank you!

**Comment by Progi1984** [comment]
Duplicate of #12556
